app/utils/encryptable_transfer.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class EncryptableTransfer
+  def transfer(encryptable, receiver, sender)
+    transfer_password = new_transfer_password
+    encryptable.encrypt(transfer_password)
+
+    encryptable.update!(
+      folder: inbox_folder(receiver),
+      sender_id: sender.id,
+      receiver_id: receiver.id,
+      encrypted_transfer_password: encrypted_transfer_password(transfer_password, receiver)
+    )
+    encryptable
+  end
+
+  def receive(encryptable, private_key, personal_team_password)
+    encryptable.decrypt_transfered(private_key)
+
+    encryptable.update!(encrypted_transfer_password: nil,
+                        receiver_id: nil,
+                        sender_id: nil)
+
+    encryptable.encrypt(personal_team_password)
+    encryptable.decrypt(personal_team_password)
```

Comment:
do we need to encrypt/decrypt here?
we're already doing this at decrypt_transfered?